src/Gui/Stylesheets/images_dark-light/checkbox_light.svg

```diff
@@ -2,66 +2,41 @@
 <!-- Created with Inkscape (http://www.inkscape.org/) -->
 
 <svg
-   xmlns:dc="http://purl.org/dc/elements/1.1/"
-   xmlns:cc="http://creativecommons.org/ns#"
-   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
-   xmlns:svg="http://www.w3.org/2000/svg"
-   xmlns="http://www.w3.org/2000/svg"
-   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
-   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
    width="30"
    height="30"
    id="svg2"
    version="1.1"
-   inkscape:version="0.92.2 (5c3e80d, 2017-08-06)"
-   sodipodi:docname="checkbox_light.svg"
-   viewBox="0 0 30 29.999999">
-  <defs
-     id="defs4" />
+   viewBox="0 0 30 29.999999"
+   sodipodi:docname="checkbox_dark.svg"
+   inkscape:version="1.2-beta (1b65182ce9, 2022-04-05)"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:dc="http://purl.org/dc/elements/1.1/">
```

Comment:
The SVGs need to be saved without inkscape header info, is what I think @benj5378 is referring to. 
In inkscape it's something like: 
Open the save dialog [File > Save As...] and in the drop down list choose Optimised SVG

Reply:
Plain SVG, not optimised XD else it will be all collapsed and stuff I believe. We usually save as Plain SVG in the repo as far as I know.